ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.js

```diff
@@ -38,15 +40,25 @@ export default class MessagesList extends Component {
     }
   }
 
+  get hasSomeActiveModals() {
+    if (!this.args.messages) return false;
```

Comment:
Following up on my previous suggestion - will this be undefined or an empty array if there are no messages? If an empty array we'll need to check for length

Reply:
Since this is the only place in this component where the array of messages is used, I wonder if we can move this logic to the router and instead pass a boolean to this component indicating whether or not there are active modals